PhysicsTools/Heppy/python/analyzers/objects/TauAnalyzer.py

```diff
@@ -84,8 +87,8 @@ def id6(tau,X):
             tau.idMVA = id6(tau, "by%sIsolationMVArun2v1DBoldDMwLT")
             tau.idMVANewDM = id6(tau, "by%sIsolationMVArun2v1DBnewDMwLT")
             tau.idCI3hit = id3(tau, "by%sCombinedIsolationDeltaBetaCorr3Hits")
-            tau.idMVAOldDMRun2 = id3(tau, "by%sIsolationMVArun2v1DBoldDMwLT")
-            tau.idMVAOldDMRun2dR03 = id3(tau, "by%sIsolationMVArun2v1DBdR03oldDMwLT")
+            tau.idMVAOldDMRun2 = id5(tau, "by%sIsolationMVArun2v1DBoldDMwLT")
```

Comment:
Warning: the encoding of tau.idMVAOldDMRun2 is not backward compatible (i.e. the cut value should be changed downstream to keep the same selection)